website/docs/cdktf/cli-reference/commands.mdx

```diff
@@ -408,10 +411,10 @@ Synthesize code when providing a custom command to execute and an output directo
 $ cdktf synth --app="npm compile && node main.js" --output="dirname"
 ```
 
-Synthesize code for an application and output the Terraform JSON configuration.
+Synthesize code for an application and output the Terraform JSON configuration for two stacks.
```

Comment:
Is this a second new feature introduced by this PR? Do we need this?
The PR title sounds only like the outputs related feature.
If we print the Terraform JSON configuration and users start to use that one, they also need to know how to handle assets and where they are located, right? I think we should add a bit more thought first, before exposing this? Or has this happened already and there's something you could link me to?

Reply:
This is not new, we had this behaviour (just for one stack and now for more), it was introduced quite a while ago: https://github.com/hashicorp/terraform-cdk/pull/104/files#r439771587

Reply:
Oh, hehe 😅